ros/src/computing/perception/detection/lib/image/librcnn/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(librcnn)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  sensor_msgs
+  std_msgs
+)
+
+find_package(OpenCV REQUIRED)
+
+###########################################CAFFE NEEDS TO BE PREVIOUSLY COMPILED####################
+##############DONT FORGET TO INSTALL fastrcnn's caffe dynamic libraries to /usr/local/lib	#### 
+set(CAFFE_PATH "$ENV{HOME}/fast-rcnn/caffe-fast-rcnn/distribute")				####
```

Comment:
Is `/distribute` necessary ? I build `fast-rcnn` following steps, then there is no `distribute` directory.

```
% git clone --recursive https://github.com/rbgirshick/fast-rcnn.git
% git submodule update --init --recursive
% cd fast-rncc
% (cd lib/ && make)
% (cd caffe-fast-rcnn && cmake . && make && make pycaffe)
```